apps/web/components/organisms/newOpinionForm/newOpinionForm.tsx

```diff
@@ -0,0 +1,52 @@
+import { Button, Card, Heading } from 'ui';
+import { useAddOpinionForm } from 'organisms/newOpinionForm/useAddOpinionForm';
+import { TextArea } from 'ui/components/organisms/textArea/textArea';
+import { ReactComponent as StarIcon } from 'components/icons/gray-star.svg';
+
+export const NewOpinionForm = () => {
```

Comment:
`OpinionForm` (more generic and could be used for opinion edition) or maybe to keep it consistent with comments, `AddOpinion` ?